spring-boot-actuator/src/main/java/org/springframework/boot/actuate/trace/TraceProperties.java

```diff
@@ -48,6 +49,12 @@
 	 */
 	private Set<Include> include = new HashSet<Include>(DEFAULT_INCLUDES);
 
+	/**
+	 * Whether {@literal Cookie} and {@literal Set-Cookie} should be excluded from
+	 * request/response headers. Defaults to {@code false}.
+	 */
+	private Boolean excludeCookies = false;
```

Comment:
I am not sure. I think I'd have a `Exclude.COOKIE` that explains what that does and apply the excludes on top of the includes or something?